general-agent-module/src/events.checker.ts

```diff
@@ -0,0 +1,26 @@
+import { Finding, FindingSeverity, FindingType, HandleTransaction, TransactionEvent } from "forta-agent";
+
+const createFinding = (alertId: string, eventSignature: string): Finding => {
+    return Finding.fromObject({
+        name: "Event Checker",
+        description: "The specified event was found",
+        alertId,
+        severity: FindingSeverity.Medium,
+        type: FindingType.Exploit,
+        metadata: {
+            EventSignature: eventSignature
+        },
+    });
+};
+
+export default function provideEventCheckerHandler(eventSignature: string, alertId: string, address: string | undefined): HandleTransaction {
+    return async (txEvent: TransactionEvent): Promise<Finding[]> => {
+        const findings: Finding[] = [];
+
+        if (txEvent.filterEvent(eventSignature, address).length > 1) {
```

Comment:
```suggestion
        if (txEvent.filterEvent(eventSignature, address).length > 0) {
```
Are you using 1 by mistake or there is an special reason for that?